monai/data/meta_tensor.py

```diff
@@ -181,30 +184,34 @@ def update_meta(rets: Sequence, func, args, kwargs) -> Sequence:
                 ret = ret.as_tensor()
             # else, handle the `MetaTensor` metadata.
             else:
-                meta_args = MetaObj.flatten_meta_objs(list(args) + list(kwargs.values()))
-                ret._copy_meta(meta_args)
+                meta_args = MetaObj.flatten_meta_objs(args, kwargs.values())  # type: ignore
+                ret._copy_meta(meta_args, deep_copy=not is_batch)
```

Comment:
this is a workaround for not deepcopying attributes when `metatensor.is_batch` flag is `True`